.github/workflows/ci.yml

```diff
@@ -0,0 +1,36 @@
+name: CI
+
+on:
+  pull_request: {}
+  push:
+    branches:
+      - master
+
+jobs:
+  tfsec:
+    name: tfsec
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: aquasecurity/tfsec-action@v1.0.3
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+
+  terraform:
+    name: Terraform
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
```

Comment:
What's the purpose of creating separate jobs to perform operations with the same data (the terraform project)? If the `tfsec` job will block the `terraform`, we could do this process in one single job with the `tfsec` as the first step. If there is no dependency on each other and the only thing that concerns us is the failure (project not safe and/or the validation), we could do these jobs in parallel.

Reply:
The jobs are not codependent, so GitHub Actions will run them in parallel by default. This behavior is documented in [this overview.](https://docs.github.com/en/actions/using-jobs/using-jobs-in-a-workflow#overview) As to what the purpose may be: not only do these jobs not have any common dependencies or dependencies on one another, they achieve different purposes. For `tfsec`, the purpose is security. For `terraform`, the purpose is to ensure the Terraform code being submitted as a pull request is valid as far as Terraform is concerned.

Later down the line, the `terraform` job will be expanded to include a `terraform plan` invocation, with the added purpose of comparing existing infrastructure to what has been declared. In and of itself, that cannot be considered a status check, as unintended changes may have been planned, but the Terraform plan is a more than valid requirement for a CI build, as it helps reviewers spot what changes would take place after a merge.